bundle/default-bundle/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+camunda.connector.polling.enabled=true
+camunda.connector.polling.interval=5000
+
+camunda.connector.webhook.enabled=true
+#spring.main.web-application-type=none
+
+# uncomment to activate inbound and adjust operate config if necessary
+#camunda.connector.polling.enabled=true
+#camunda.connector.polling.interval=5000
```

Comment:
Why do we leave the commented properties in as they are active above?

Reply:
Good point - I removed those